### PR TITLE
Rigged Hands graduation part 2

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Graduates rigged hands from experimental to mainline by moving them out of the directories. Tests enabled

Followup to #8714

Changes
Fixes: #8669 modulo file refactor

### CODE REVIEW

File: Documentation/README_RiggedHandVisualizer.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Rigged Hand Visualizer [Experimental]
+# Rigged Hand Visualizer
```
Comment (CDiaz-MS) [inline Documentation/README_RiggedHandVisualizer.md:1]: It would be awesome if we could make this documentation more tutorial-esque, with screen shots of the inspector and property descriptions, but that would probably be a different PR.

**Review by CDiaz-MS — APPROVED** [review]

File: Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft bones.prefab.meta [hunk 2]
```diff
@@ -1,6 +1,6 @@
 fileFormatVersion: 2
-guid: 89306ab5ab6988249a8aa53c429567a2
-TextScriptImporter:
+guid: 3cfb75756a9f6474481612e89b7931b5
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft bones.prefab.meta:2]: I don't see the corresponding prefab for this meta in this PR. Should it be added or this removed?

File: Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.RiggedHandVisualizer
+namespace Microsoft.MixedReality.Toolkit.Input
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs:11]: It seems like most of the other scripts in the SDK/Features/UX folder are in the `.UI` namespace. Should this be in that one too, or moved to a different folder?

File: Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerVisualizationProfile.asset [hunk 4]
```diff
@@ -51,3 +51,24 @@ MonoBehaviour:
...
+    handedness: 1
+    useDefaultModel: 0
+    defaultModelMaterial: {fileID: 0}
+    overrideModel: {fileID: 61451597023024772, guid: 6080fb9e3b49fda449e3d70daf6d2254,
+      type: 3}
+    controllerVisualizationType:
+      reference: Microsoft.MixedReality.Toolkit.Input.RiggedHandVisualizer, Microsoft.MixedReality.Toolkit.SDK
+  - description: Generic Joystick Controller
+    controllerType:
+      reference: Microsoft.MixedReality.Toolkit.Input.UnityInput.GenericJoystickController,
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerVisualizationProfile.asset:66]: Should this be OculusHand?

File: Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs (hunk 3 above)
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/UX/Scripts/RiggedHandVisualizer/RiggedHandVisualizer.cs:11]: Our BaseHandVisualizer is in the .Input namespace, so I thought it would make sense to have this be in there too.

File: Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft bones.prefab.meta (hunk 2 above)
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/UX/Prefabs/RiggedHandVisualizer/RiggedHandLeft bones.prefab.meta:2]: need to refactor out a bunch of stuff that popped up from the version diffs, processing now
